### PR TITLE
document driver-wd loadPage()

### PR DESCRIPTION
**Pull request by dbushong**

### CODE REVIEW

File: api/wd/index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Asserts that the most recent response status code is `statusCode`.
 
 ```js
 browser
-  .navigateTo('/products');
+  .loadPage('/products');
   .assertStatusCode(200);
```
Comment (jkrems) [inline api/wd/index.md]: Should this line be removed now?

**Comment by dbushong** [comment]
How's that?

File: api/wd/index.md (hunk 1 above)
Comment (jkrems) [inline api/wd/index.md]: Oh, that makes sense. :D

**Comment by jkrems** [comment]
👍

**Pull request merged**